src/main/java/seedu/academydirectory/model/student/Sortable.java

```diff
@@ -0,0 +1,62 @@
+package seedu.academydirectory.model.student;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Arrays;
+import java.util.List;
+
+public abstract class Sortable {
```

Comment:
Also I think since the only thing u can sort is Information, so maybe the Sortable and SortableInformation can be merged into one class that is just SortableInformation?

Reply:
Agreed, will combine them and do away with SortableWrapper